pgcli/packages/sqlcompletion.py

```diff
@@ -132,6 +132,9 @@ def suggest_special(text):
             return [{'type': 'schema'},
                     {'type': rel_type, 'schema': []}]
 
+    if cmd == '\\n':
```

Comment:
Can we make this 

```
if cmd in ('\\n', '\\ns', '\\nd'):
```

That way it'll suggest the names for all those special commands?